test/models/parse_utils_test.rb

```diff
@@ -50,21 +50,35 @@ def test_cell_ranger_expression_parse
   # test that sparse matrix parsing validates coordinate matrix sort order correctly
   def test_sparse_matrix_sort_check
     puts "#{File.basename(__FILE__)}: #{self.method_name}"
-    @study = Study.find_by(name: 'Negative Testing Study')
+
+    # create a test study and add matrix files and bundle
     user = User.first
-    matrix = @study.study_files.by_type('MM Coordinate Matrix').first
-    genes = @study.study_files.by_type('10X Genes File').first
-    barcodes = @study.study_files.by_type('10X Barcodes File').first
+    study = Study.create(name: 'Negative Testing Study', firecloud_project: ENV['PORTAL_NAMESPACE'], user_id: user.id)
+    user = User.first
+    mm_coord_file = StudyFile.create!(name: 'GRCh38/test_bad_matrix.mtx', upload: File.open(Rails.root.join('test', 'test_data', 'GRCh38', 'test_bad_matrix.mtx')),
+                                      file_type: 'MM Coordinate Matrix', study_id: study.id)
+    genes_file = StudyFile.create!(name: 'GRCh38/test_genes.tsv', upload: File.open(Rails.root.join('test', 'test_data', 'GRCh38', 'test_genes.tsv')),
+                                   file_type: '10X Genes File', study_id: study.id, options: {matrix_id: mm_coord_file.id.to_s})
+    barcodes = StudyFile.create!(name: 'GRCh38/barcodes.tsv', upload: File.open(Rails.root.join('test', 'test_data', 'GRCh38', 'barcodes.tsv')),
+                                 file_type: '10X Barcodes File', study_id: study.id, options: {matrix_id: mm_coord_file.id.to_s})
+
+    study_file_bundle = study.study_file_bundles.build(bundle_type: mm_coord_file.file_type)
+    bundle_payload = StudyFileBundle.generate_file_list(mm_coord_file)
+    study_file_bundle.original_file_list = bundle_payload
+    study_file_bundle.save!
+
+    # gotcha for refreshing matrix file after creating bundle to pick up association
+    mm_coord_file.reload
 
     begin
       puts 'Parsing 10X incorrectly sorted matrix...'
-      ParseUtils.cell_ranger_expression_parse(@study, user, matrix, genes, barcodes, {skip_upload: true, sync: true})
+      ParseUtils.cell_ranger_expression_parse(study, user, mm_coord_file, genes_file, barcodes, {skip_upload: true, sync: true})
```

Comment:
Consistent parameter naming:

```suggestion
      ParseUtils.cell_ranger_expression_parse(study, user, mm_coord_file, genes_file, barcodes_file, {skip_upload: true, sync: true})
```